packages/client/src/components/Disclaimer.tsx

```diff
@@ -1,23 +1,22 @@
-import { Alert, Paragraph } from "@amsterdam/asc-ui";
+import { CompactThemeProvider, Paragraph } from "@amsterdam/asc-ui";
 import React from "react";
+import { useTranslation } from "react-i18next";
 
 import { DISCLAIMER_TEXT } from "../utils/test-ids";
-import ContactSentence from "./ContactSentence";
 
-const Disclaimer: React.FC = () => (
-  <Alert heading="Let op">
-    <Paragraph data-testid={DISCLAIMER_TEXT}>
-      De vergunningcheck is nog in ontwikkeling. Hierdoor kunnen wij nog geen
-      zekerheid bieden dat de uitkomst correct is. Ook is de informatie nog niet
-      voor iedereen goed te lezen of te beluisteren. Wilt u iets zeker weten of
-      wilt u meer informatie?{" "}
-      <ContactSentence
-        eventName=""
-        link={false}
-        openingSentence={"Bel dan de gemeente op"}
-      />
-    </Paragraph>
-  </Alert>
-);
+const Disclaimer: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <CompactThemeProvider>
+      <Paragraph
+        data-testid={DISCLAIMER_TEXT}
+        forwardedAs="em"
+        gutterBottom={0}
+      >
+        {t("outcome.disclaimer")}
```

Comment:
No more ContactSentence?

Reply:
No, ContactSentence is removed from the disclaimer text